opentracing/src/main/java/opentracing/Span.java

```diff
@@ -25,16 +25,7 @@
   /**
    * Suitable for serializing over the wire, etc.
    */
-  TraceContext traceContext();
-
-  /**
-   * Denotes the beginning of a subordinate unit of work.
-   *
-   * @param operationName name of the operation represened by the new span from the perspective of
-   * the current service.
-   * @return a new child Span in "started" state.
-   */
-  Span startChild(String operationName);
+  SpanContext childContext();
```

Comment:
>  From various comments on this thread, including the one quoted above, it seems your SpanContext is not meant to perform either of these functions.

No. SpanContext solves both those functions.
Can you elaborate which comments allude otherwise?